bin/oneoff/backfill_data/channel_tokens_storage_app_id.rb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+
+# This script populates the storage_apps_id column of the channel_tokens table.
+
+require_relative '../../../dashboard/config/environment'
+require_relative '../../../shared/middleware/helpers/storage_id'
+
+ChannelToken.find_each do |channel_token|
+  storage_app_id, _channel_id = storage_decrypt_channel_id(channel_token.channel)
+  channel_token.storage_app_id = storage_app_id
+  channel_token.save!
+end
```

Comment:
perhaps now is a good time to ask, what are your thoughts on how we'd go about renaming the storage_apps table in the event that we decided to do it? Is it possible to set up an table name alias of some kind in the DB, so that we don't have to do the code change in sync with the DB change?